packages/react/src/components/Storage/FileUploader/UploadButton/index.tsx

```diff
@@ -1,41 +1,22 @@
 import React from 'react';
 import { translate } from '@aws-amplify/ui';
 import { UploadButtonProps } from '../types';
-import { Button, VisuallyHidden } from '../../../../primitives';
+import { Button } from '../../../../primitives';
 
 export function UploadButton({
```

Comment:
Now this is just a button, and we could probably add the props below directly in FileUploader and remove this component altogether

Reply:
I was chatting with @dbanksdesign , I think for now we want to keep it, so customers can completely override the component using our component overrides. If they wanted to use a different design system, they can still do the override.

Reply:
@ErikCH Think you misunderstood here. `UploadButton` is now just wrapping a `Button`, so you can just assign the default value of `components.UploadButton` [here](https://github.com/aws-amplify/amplify-ui/blob/fileuploader/main/packages/react/src/components/Storage/FileUploader/FileUploader.tsx#L36) to `Button`

Reply:
Thanks, makes sense. Updated, and pushed to `fileuploader/main` I deleted the `UploadButton` but left in the override [6734314](https://github.com/aws-amplify/amplify-ui/pull/3028/commits/673431409ba8cc1893bd4da7ff5ea095586c4d5c)